docs/api/navigators/DrawerNavigator.md

```diff
@@ -79,7 +79,12 @@ this.props.navigation.navigate('DrawerClose'); // close drawer
 DrawerNavigator(RouteConfigs, DrawerNavigatorConfig)
 ```
 
-### Drawer Navigator Options
+### RouteConfigs
+
+The route configs object is a mapping from route name to a route config, which tells the navigator what to present for that route, see [example](https://github.com/coodoo/react-navigation/blob/master/docs/api/navigators/StackNavigator.md#routeconfigs) from `StackNavigator`.
```

Comment:
This link should point to the main repo.

Also, isn't there someplace else where route configs are documented? We should link there